jl4-cli/examples/not-ok/tests/parse-error1.ep.golden

```diff
@@ -0,0 +1,13 @@
+Info | updateFileDiagnostics published different from new diagnostics - file diagnostics:
+File:<<file>>
```

Comment:
`<<file>>` is weird? Why would the filename not be available? If we ever have multi-module tests, this should be printed anyway.

Reply:
it is only like that for the goldens where the full filename includes directories that have different names depending on the machine.

Reply:
Fair enough, although I think we dropped the paths and kept the basenames in the past. As I said, it's possible that in the not too distant future we might have tests involving multiple modules, so in which file the error is can still be relevant.